src/featurization_utils/src/README.md

```diff
@@ -0,0 +1,16 @@
+## Texture Features
+| Feature                                         | Typical Range                               | Interpretation                                                                                                                      |
+| ----------------------------------------------- | ------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------- |
+| **Angular Second Moment (Energy / Uniformity)** | **0 → 1**                                   | Texture **uniformity**. Higher = more uniform/smoother.                                                                             |
```

Comment:
Seeing these values made me wonder: could this be used for schema validation? Schema validation is a tool we can implement to help check that data fields and values abide certain expecations. A common tool used for this today is `jsonschema`. It might seem a bit odd to use that library because the data won't necessarily be JSON, but it could be useful to help define a schema and ranges. Another helpful tool in this space is [Pandera](https://pandera.readthedocs.io/en/stable/) which builds on this idea using Pydantic and Pandas idioms.